libcloudforensics/providers/gcp/internal/metrics.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Google Cloud Metrics functionality."""
+
+import datetime
+from typing import TYPE_CHECKING, Dict
+
+from libcloudforensics.providers.gcp.internal import common
+
+if TYPE_CHECKING:
+  import googleapiclient
+
+
+class GoogleCloudMetrics:
```

Comment:
I believe we should call this cloud monitoring and make the name consistent in all the changed files, since the official name in public docs is Cloud Monitoring API? what do you think?